layouts/states/register-by-mail.html

```diff
@@ -0,0 +1,50 @@
+<div class="usa-grid page-inner">
+  <header>
+    <h1 class="usa-display">Register to vote</h1>
+  </header>
+  <!-- Steps BEGIN -->
+  <div class="graphic-list">
+    <img src="{{ .Site.BaseURL }}assets/img/step-1.png">
+    <h4 class="usa-graphic-list-heading">Step 1</h4>
+    <p>Fill out your voter registration.</p>
+    <img src="{{ .Site.BaseURL }}assets/img/step-2.png">
+    <h4 class="usa-graphic-list-heading">Step 2</h4>
+    <p>Mail it to your state.</p>
+    <img src="{{ .Site.BaseURL }}assets/img/step-3.png">
+    <h4 class="usa-graphic-list-heading">Step 3</h4>
+    <p>Vote in the upcoming election!</p>
+  </div>
+  <!-- Steps END -->
+
+  <!-- Download form BEGIN -->
+  <b>
+    {{ range $key, $form := $.Site.Data.registration_forms }}
+      {{ if gt $form.weight 0 }}
+      <a href="{{ $form.file_path }}">Download the registration form (PDF)</a>
+      {{ end }}
+    {{ end }}
```

Comment:
Currently, this logic in the `register-by-mail` layout is biased towards the English language.